design/object-bucket/object-bucket.md

```diff
@@ -6,29 +6,26 @@ Enable Rook-Ceph Object users the ability to request buckets in an object store
 
 ## Glossary
 
-- Admin: A Kubernetes cluster administrator with RBAC permissions to instantiate and control access to cluster storage. The admin will use existing Rook-Ceph CRDs to create Ceph Object Stores as well as StorageClasses to make them accessible to users.  
+- Admin: A Kubernetes cluster administrator with RBAC permissions to instantiate and control access to cluster storage. The admin will use provision object stores via CephObjectStore custom resources and create StorageClasses to enable user access.  
 - User: A Kubernetes cluster user with limited permissions, typically confined to CRUD operations on Kubernetes objects within a single namespace.  The user will request Ceph Object buckets by instantiating ObjectBucketClaims.
-- ObjectBucketClaim: a namespaced, user facing Custom Resource Definition representing a request for a dynamically created bucket (similar in use to PersistentVolumeClaims).  ObjectBucketClaims will reference a StorageClass whose _provisioner:_ field contains the name of an object store provisioner capable of dynamically provisioning buckets.  ObjectBucketClaims serve to abstract the underlying object store provider from the user.
+- ObjectBucketClaim: a namespaced, user facing Custom Resource Definition representing a request for a dynamically created bucket (similar in use to PersistentVolumeClaims).  ObjectBucketClaims reference a StorageClass whose _provisioner:_ field contains the name of an object store provisioner capable of dynamically provisioning buckets.  ObjectBucketClaims serve to abstract the underlying object store provider from the user.
 - ObjectBucket: a non-namespace, admin facing Custom Resource Definition representing the fulfillment of a user request for a dynamically created bucket in an existing object store (similar in use to PersistentVolumes).  ObjectBuckets serve to provide discrete information about the bucket and object store provider to admins.
 
 ## Goals
 
 - Provide Rook-Ceph users the ability to dynamically provision object buckets via the Kubernetes API
 - Enable cluster admin control over bucket creation and object store access via the Kubernetes API
-- Restrict users from creating buckets with their access keys outside the view of Kubernetes (e.g. via `s3cmd` CLI tool)
+- Restrict users from creating buckets with automation-generated access keys outside of the Kubernetes API (e.g. via `s3cmd` CLI tool)
 - Utilize a familiar and deterministic pattern when injecting bucket connection information into workload environments
 - Ensure the consuming Pod will not be started until the bucket has been created
```

Comment:
Is another goal here to promote the definition of a generic bucket claim and bucket APIs? Or will that goal be a different PR (somewhere)?

Reply:
Not IMO - a proposal to Rook-Ceph shouldn't come with exterior political motives.

Reply:
we will add a doc/proposal for a the generic aspects of this pr. This will temporarily live in our yard-turkey repo until we can find a better home...